scripts/docker-quick-start/sql/apolloconfigdb.sql

```diff
@@ -61,7 +61,7 @@ DROP TABLE IF EXISTS `AppNamespace`;
 
 CREATE TABLE `AppNamespace` (
   `Id` int(10) unsigned NOT NULL AUTO_INCREMENT COMMENT '自增主键',
-  `Name` varchar(32) NOT NULL DEFAULT '' COMMENT 'namespace名字，注意，需要全局唯一',
+  `Name` varchar(500) NOT NULL DEFAULT '' COMMENT 'namespace名字，注意，需要全局唯一',
```

Comment:
Shall we also change the limitation in other places like [valdr.js](https://github.com/apolloconfig/apollo/blob/3440475ca0ad59fadc23f583c9fc36594dda1318/apollo-portal/src/main/resources/static/scripts/valdr.js#L66) so that users can create a namespace with length longer than 32?

Reply:
I ignored that there is a layer of verification in the front end, and the input restrictions are considered to be redesigned so that the operation and maintenance personnel can adjust it. I'll close this pr first